README.rst

```diff
@@ -160,6 +160,65 @@ In .pre-commit-config.yaml:
 * ``.*\.js`` excludes all .js files in all directories.
 * ``\..*`` excludes all hidden files.
 
+``tech-review`` setup
+---------------------
+
+These are the default values for the arguments of the tech-review hook:
+
+* --author_maint_name=ANSYS, Inc.
+* --author_maint_email=pyansys.core@ansys.com
+* --start_year=2024 (or the current year)
+* --license=MIT
+* --url=https://github.com/ansys/{repo-name}, replacing `repo-name` with the name of the repository
+
+The `--author_maint_name` is the name of the author and maintainer in the `pyproject.toml` file.
+By default, it is "Ansys, Inc.".
+
+The `--author_maint_email` is the email of the author and maintainer in the `pyproject.toml` file.
+By default, it is "pyansys.core@ansys.com".
+
+The `--start_year` argument is the start year of your repository. If left blank, the default is
+the current year.
+
```

Comment:
Can't we retrieve this automatically from the git history? You should be able to search for the information of the first commit. And otherwise default to the current year. And if passed by value, then we take that value as is.

We might want to replicate this same logic on the license hook.

Reply:
Sure, I didn't know you could do that. I'll try it out